packages/@aws-cdk/aws-backup/lib/selection.ts

```diff
@@ -0,0 +1,154 @@
+import iam = require('@aws-cdk/aws-iam');
+import { Construct, IResource, Resource } from '@aws-cdk/core';
+import { CfnBackupSelection } from './backup.generated';
+import { IBackupPlan } from './plan';
+
+/**
+ * Represents a Backup Selection.
+ */
+export interface IBackupSelection extends IResource {
+  /**
+   * The name of the selection.
+   * @attribute
+   */
+  readonly backupSelectionName: string;
+  /**
+   * Optional IAM Role to use for accessing the resources
+   * @attribute
+   */
+  readonly role?: iam.IRole;
+  /**
+   * The Backup Plan to relate the selections to.
+   * @attribute
+   */
+  readonly backupPlan: IBackupPlan;
+  /**
+   * A string representation of the backup plan ID.
+   * @attribute
+   */
+  readonly backupSelectionBackupPlanId: string;
+  /**
+   * A string representation of the backup selection ID.
+   * @attribute
+   */
+  readonly backupSelectionSelectionId: string;
+  /**
+   * Optional resources' ARN to select.
+   * @attribute
+   */
+  readonly resources?: string[];
+}
+
+abstract class BackupSelectionBase extends Resource implements IBackupSelection {
+  public abstract readonly backupSelectionName: string;
+  public abstract readonly backupPlan: IBackupPlan;
+  public abstract readonly backupSelectionBackupPlanId: string;
+  public abstract readonly backupSelectionSelectionId: string;
+  public abstract readonly role?: iam.IRole;
+  public abstract readonly resources?: string[];
+}
+
+/**
+ * Backup selection attributes.
+ */
+export interface BackupSelectionAttributes {
+  /**
+   * A string representation of the backup selection.
+   */
+  readonly backupSelectionName: string;
+  /**
+   * A backup plan object.
+   */
+  readonly backupPlan: IBackupPlan;
+  /**
+   * The backup selection's selection Id.
+   */
+  readonly backupSelectionSelectionId: string;
+}
+
+/**
+ * Represents the backup selection properties.
+ */
+export interface BackupSelectionProps {
+  /**
+   * The name of the selection resource.
+   */
+  readonly backupSelectionName: string;
+  /**
+   * The backup plan to assign this plan to.
+   */
+  readonly backupPlan: IBackupPlan;
+  /**
+   * Optional IAM Role to use for selecting and interacting with the AWS resources to backup.
+   * @default IAM Role based from the AWS Backup service.
+   * @see https://docs.aws.amazon.com/aws-backup/latest/devguide/iam-service-roles.html
+   */
+  readonly role?: iam.IRole;
+  /**
+   * A list of ARNs to backup.
+   * @default to no resources.
+   */
+  readonly resources?: string[];
+}
+
+/**
+ * Backup resource selection.
+ */
+export class BackupSelection extends BackupSelectionBase {
+  /**
+   * Imports a backup selection based from the given attributes.
+   */
+  public static fromBackupSelectionAttributes(scope: Construct, id: string, attrs: BackupSelectionAttributes): IBackupSelection {
```

Comment:
Not sure we need to be bale to import a backup selection. There's nothing you can do with it once imported.

Reply:
I'll try to delete it, but if I remember properly I had to add it because the lint was complaining that I should add it.